setup.json

```diff
@@ -1,221 +1,220 @@
 {
-  "name": "aiida-core",
-  "version": "1.0.0a4",
-  "url": "http://www.aiida.net/",
-  "license": "MIT License",
-  "author": "The AiiDA team",
-  "author_email": "developers@aiida.net",
-  "include_package_data": true,
+  "name": "aiida-core", 
```

Comment:
hm... what is adding the space here? is this the `validate_consistency.py` script?

Reply:
Yes this was done automatically, I think it is the json dumper that is being used

Reply:
we could use [this](https://stackoverflow.com/a/16311587)

Reply:
If it bothers you, I am fine with adding it

Reply:
yes, please!

Reply:
Everything else ok? Like to do all necessary changes in one go, so we don't have to run too many builds

Reply:
I looked through everything and did not see any other problems

Reply:
Done and done